controllers/drplacementcontrol_controller.go

```diff
@@ -263,7 +263,9 @@ func DRClusterPredicateFunc() predicate.Funcs {
 			return false
 		},
 		DeleteFunc: func(e event.DeleteEvent) bool {
-			return false
+			log.Info("Delete event")
+
+			return true
```

Comment:
This does not work, looks like FilterDRCluster is filtering only drcluster that are the failover clusters for some drpcs, including skiping for metro dr. This code is complex and we don't have enough time to modify it now.

I'll work on this later.